android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/SpeakerViewHolder.java

```diff
@@ -50,9 +54,35 @@ public SpeakerViewHolder(View itemView, Context context) {
         //Attach onClickListener for ViewHolder
         itemView.setOnClickListener(view -> {
             String speakerName = speaker.getName();
+                       
+            Bundle bundle = null;
+            
             Intent intent = new Intent(this.context, SpeakerDetailsActivity.class);
             intent.putExtra(Speaker.SPEAKER, speakerName);
-            this.context.startActivity(intent);
+            
+            if(this.context instanceof Activity){
```

Comment:
Don't use instanceOf 

The idea is that we want to make it so that if someone changes the code in future, and the class is not an Activity, this should give a syntax error. 

Use straightforward logic here please

Reply:
@iamareebjamal  Should this code should be able to handle any class or only Activity class ?

Reply:
@iamareebjamal But Sir ''makescenetransition" takes an activity as its first entry , like it has to been an activity!